open-sphere-base/core/src/main/java/io/opensphere/core/net/NetworkConfigurationOptionsProvider.java

```diff
@@ -146,6 +146,8 @@ else if (myUseManualProxyButton.isSelected())
             myNetworkConfigurationManager.setProxy("", -1);
             myNetworkConfigurationManager.setProxyExclusions("");
         }
+
+        ((NetworkConfigurationManagerImpl)myNetworkConfigurationManager).notifyChanged();

```

Comment:
I don't like having to cast to impl.  Why did we have to move notifyChanged to be called here?

Reply:
I want it to be clear that it's only called once when the apply button is hit. Before it was being called once for every single change (setProxy, setProxyExclusions, etc.) within the network configuration manager logic - I don't think that's necessary. I did change it so that a cast is not necessary - let me know if there's still an issue and I can fix it.